.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: node_js
+cache:
+  yarn: true
+notifications:
+  email: false
+node_js:
+  - '8'
+before_install:
+  - cd taui
+  - yarn global add codecov
+scripts:
+  - yarn test -- --coverage --coverage-paths src/**/*.js
+  - codecov
+  - yarn run build
```

Comment:
In `package.json` it looks like the `test` script runs `build` for us:

https://github.com/azavea/echo-locator/blob/9bc408bc30745b7709cb21a3740801809dc0e3a1/taui/package.json#L13

I'd say we can probably safely remove this line.

```suggestion
```